spec/models/manageiq/providers/azure/cloud_manager/metrics_capture_spec.rb

```diff
@@ -4,58 +4,71 @@
   let(:ems)      { FactoryBot.create(:ems_azure) }
   let(:vm)       { FactoryBot.build(:vm_azure, :ext_management_system => ems, :ems_ref => "my_ems_ref") }
   let(:group)    { FactoryBot.build(:azure_resource_group) }
+  let(:armrest_environment) { double('Armrest environment', :resource_url => 'http://example.com') }
```

Comment:
I'd like people to use `instance_double` over `double` so that we can be sure our mocks are valid.

Reply:
Dan, I don't think we should test `azure-armrest` gem's internals here. This line can be replaced with `OpenStruct.new(resource_url: 'something')` here with exactly the same result. It's just a stub.